halconfig/orca.yml

```diff
@@ -1,43 +1 @@
 # halconfig
```

Comment:
why does this file still need to exist? Does `halyard` get mad otherwise?

Reply:
I'm 99% sure that it will get mad...though I'll be honest that I didn't actually check.  If these changes look good and become my plan for all the services, I'll double-check if we can actually completely delete the files.